var/spack/repos/builtin/packages/py-fenics-dolfinx/package.py

```diff
@@ -49,20 +51,23 @@ class PyFenicsDolfinx(PythonPackage):
     depends_on("fenics-dolfinx@0.1.0", when="@0.1.0")
 
     depends_on("fenics-basix@main", type=("build", "link"), when="@main")
+    depends_on("fenics-basix@0.6.0:0.6", type=("build", "link"), when="@0.6.0:0.6")
```

Comment:
I don't see where basix is required by this version

Reply:
It's in the `CMakeLists.txt` that builds the wrapper interface. Versioning is a bit sloppy in the `CMakeLists.txt` file.